libsolidity/interface/CompilerStack.cpp

```diff
@@ -631,16 +631,17 @@ vector<string> CompilerStack::contractNames() const
 	return contractNames;
 }
 
-string const CompilerStack::lastContractName() const
+string const CompilerStack::lastContractName(std::optional<std::string> const& _sourceName) const
```

Comment:
At least for string you can get rid off `std`
```suggestion
string const CompilerStack::lastContractName(optional<string> const& _sourceName) const
```